src/py_counter.py

```diff
@@ -7,47 +7,78 @@
 class PyCount:
     """Class for the python counter."""
 
-    def __init__(self):
+    def __init__(self, filename):
         """Declaring the self variable."""
-        self.search = cst.parse_module(
-            open("input/sample_file.py").read()  # pylint: disable=R1732,W1514
-        )
+        with open(filename, 'r') as file: # pylint: disable=W1514
+            self.search = cst.parse_module(file.read())  # pylint: disable=R1732,W1514
```

Comment:
W1514 should be resolved; if you're not sure how, this [page](https://www.kianmeng.org/2021/11/w1514-using-open-without-explicitly.html) can help. You should not need to disable R1732, as you are using `with` now.

Reply:
Please address this in the next sprint.

Reply:
@caldwella2